backend/src/v2/component/launcher_v2.go

```diff
@@ -820,6 +820,9 @@ func getPlaceholders(executorInput *pipelinespec.ExecutorInput) (placeholders ma
 		placeholders[key] = parameter.OutputFile
 	}
 
+	// Add workspace path placeholder
+	placeholders["{{$.workspace_path}}"] = WorkspaceMountPath
```

Comment:
I was originally thinking that this logic would live in the driver and be another switch statement here:
https://github.com/kubeflow/pipelines/blob/a870b1a325dae0c82c8b6f57941468ee1aea960b/backend/src/v2/driver/resolve.go#L387-L406

I think your approach of using `getPlaceholders` may enable more scenarios so that any reference to `{{$.workspace_path}}` will be replaced, not just input parameters. So I think it's better than what I had in mind. :smile: